src/main/java/it/pagopa/interop/probing/eservice/registry/updater/model/Eservice.java

```diff
@@ -49,57 +30,38 @@
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
-/**
- * The Class Eservice.
- */
 @Entity
 @Table(name = "eservices", uniqueConstraints = @UniqueConstraint(columnNames = { "eservice_id", "version_id" }))
 @TypeDef(name = "basePathType", typeClass = CustomStringArrayType.class)
-
-/**
- * To string.
- *
- * @return the java.lang. string
- */
 @Data
-
-/**
- * Instantiates a new eservice.
- */
 @NoArgsConstructor
 public class Eservice implements Serializable {
```

Comment:
Use Interface and create an Abstract Entity to manage common fields and listener